include/subgroup/tile/impl/op_function.hpp

```diff
@@ -461,6 +461,37 @@ vnni_transform(T_dst &dst, T_src &src) {
     dst.reg = reg_dst;
 }
 
+/// @brief  Converts tiled layout to transpose_tiled layout.
+///
+/// @tparam T Is the tile data type.
+/// @param mat_Acc Is the reference of the tile object.
+/// @return No return, update the data in-place.
+template <typename T>
+__XETLA_API typename std::enable_if_t<T::register_layout
+        == reg_layout::transpose_tiled>
```

Comment:
It should work for B mat from col-major global memory.

We tried the following case in `tests/unit/tile_load_store/main.cpp`
```c++
    cl::sycl::nd_range<1> nd_range({1}, {1});
    auto result_validate
            = std::bind(tile_load_store_result_validate<half, false, true>, _1,
                    _2, _3, 128, 64, 32, 32, 0);
    kernel_run<half,
            tile_load_store_func<half, 128, 64, 128, 32, 32, 16, 16, false,
                    true, 64, gpu_arch::Igpu>>(nd_range, result_validate);
```